circuits/circuits/sig.circom

```diff
@@ -0,0 +1,44 @@
+pragma circom 2.1.5;
+
```

Comment:
I think we should be importing `circom-ecdsa` here rather than in `send.circom` etc. I know circom's import system allows imports at the top level, we should probably keep the import close to where we use the thing, like in normal languages.

Reply:
will do this in the nova send PR to avoid conflicts.